src/ethstorage.js

```diff
@@ -134,21 +134,41 @@ export class EthStorage {
         return ethers.getBytes(data);
     }
 
-    async putBlobs(number, data) {
+    async putBlobs(data) {
```

Comment:
The name should be consistent with “write.” Additionally, you may need to pass “keys” as a parameter.